languages/js/src/helpers.ts

```diff
@@ -225,16 +226,15 @@ function parseExternalIsa(event: unknown): QueryEvent {
  */
 function parseExternalIsaWithPath(event: unknown): QueryEvent {
   if (!isObj(event)) throw new Error();
-  const {
-    call_id: callId,
-    base_tag: baseTag,
-    path,
-    class_tag: classTag,
-  } = event;
+  const { call_id: callId, base_tag: baseTag, class_tag: classTag } = event;
+  const path: string[] = (event.path as PolarTerm[]).map(x => {
+    const val = x.value;
+    if (!isPolarStr(val)) throw new Error();
+    return val.String;
```

Comment:
I think this would be the only place we go from Polar -> JS w/o going through `Host#toJs`. Do you like this a lot more than changing the type of `ExternalIsaWithPath.path` to `PolarTerm[]` and then mapping those PolarTerms (all of which will be Strings) in `Host#isaWithPath`? Basically what we do here: https://github.com/osohq/oso/blob/3f45dcd9561ce148cfb79f10fccbe0a5772f36f7/languages/js/src/Host.ts#L302

Reply:
No, I'm into that idea, one minute

Reply:
ok this is done, good idea!